### PR TITLE
Add GET CONFIG serial command

### PR DESCRIPTION
**Pull request by loucass003**
Implement Serial Command ``GET CONFIG``

as mentionned in https://github.com/SlimeVR/SlimeVR-Tracker-ESP/issues/6

image

### CODE REVIEW

**Review by TheDevMinerTV — APPROVED** [review]

File: src/serial/serialcommands.cpp [hunk 1]
```diff
@@ -64,6 +68,42 @@ namespace SerialCommands {
...
+
+        if (parser->equalCmdParam(1, "CONFIG")) {
+            String str =
+                "BOARD=%d\n" 
+                "IMU=%d\n"
+                "SECOND_IMU=%d\n"
+                "IMU_ROTATION=%f\n"
+                "SECOND_IMU_ROTATION=%f\n"
+                "BATTERY_MONITOR=%d\n"
+                "BATTERY_SHIELD_RESISTANCE=%d\n"
```
Comment (unlogisch04) [inline src/serial/serialcommands.cpp:80]:
```suggestion
                "BATTERY_SHIELD_RESISTANCE=%d\n"
                "BATTERY_SHIELD_R1=%d\n"
                "BATTERY_SHIELD_R2=%d\n"
```

File: src/serial/serialcommands.cpp [hunk 2]
```diff
@@ -64,6 +68,42 @@ namespace SerialCommands {
...
+
+            Serial.printf(
+                str.c_str(),
+                BOARD,
+                IMU,
+                SECOND_IMU,
+                IMU_ROTATION,
+                SECOND_IMU_ROTATION,
+                BATTERY_MONITOR,
+                BATTERY_SHIELD_RESISTANCE,
```
Comment (unlogisch04) [inline src/serial/serialcommands.cpp:97]:
```suggestion
                BATTERY_SHIELD_RESISTANCE,
                BATTERY_SHIELD_R1,
                BATTERY_SHIELD_R2,
```

**Review by unlogisch04 — CHANGES_REQUESTED** [review]
Can you add BATTERY_SHIELD_R1 and BATTERY_SHIELD_R2 too?

**Review by Eirenliel — APPROVED** [review]

**Comment by loucass003** [comment]
@unlogisch04 .... you could have checked your suggestions worked at least.

also why do we need that. those are not set in the user space

**Comment by loucass003** [comment]
im fixing it

**Comment by loucass003** [comment]
``also why do we need that. those are not set in the user space`` nevermind didnt see those

**Comment by unlogisch04** [comment]
Sorry, did not think that this would break something. And not sure how to check on Web.

**Comment by loucass003** [comment]
@Eirenliel Everything is ready to merge

**Review by unlogisch04 — APPROVED** [review]

**Pull request merged**